doc/release-notes-32944-28710-32438-31250.md

```diff
@@ -0,0 +1,12 @@
+# Legacy Wallet Removal
+
+BDB legacy wallets can no longer be created or loaded. They can be migrated to
+the new descriptor wallet format. Refer to the `migratewallet` RPC for more
+details.
+
+The legacy wallet removal drops redundant options in the bitcoin-wallet tool,
+such as `-withinternalbdb`, `-legacy`, and `-descriptors`. Moreover, the
```

Comment:
Nice touch sorting the RPCs by alphabetical order; maybe do the same for these options.